pkgs/applications/misc/keepassx/community.nix

```diff
@@ -65,6 +65,7 @@ stdenv.mkDerivation rec {
   ];
 
   cmakeFlags = [
+    "-DCMAKE_BUILD_TYPE=Release"
     "-DKEEPASSXC_BUILD_TYPE=Release"
```

Comment:
Why do we need both?

Reply:
As far as I can see from the code, if `-DKEEPASSXC_BUILD_TYPE=Release` is not set, then `CMakelists.txt` may append `-snapshot` to the version string and it may show a warning saying that it's a pre-release.

It appears the above flag is orthogonal to `-DCMAKE_BUILD_TYPE=Release`, which changes the compilation flags.